README-translated/README-Korean.md

```diff
@@ -1,10 +1,27 @@
-# The Turing Way
+<a href="https://book.the-turing-way.org/welcome.html"><img src="book/website/figures/logo/logo.jpg?raw=true)" width="180" align="Right" /></a>
```

Comment:
I think you might have accidentally merged changes from the Korean readme file here!